app/templates/article.hbs

```diff
@@ -0,0 +1,4 @@
+<div class="col-md-4 article-show">
+  <h3>{{model.title}}</h3>
+  <p>{{model.abstract}}</p>
+</div>
```

Comment:
We may need to think   about our data and visualization more. So far the articles sidebar is doing the same as the article show page in content quality.